cmd/completion.go

```diff
@@ -1,67 +1,141 @@
 package cmd
 
 import (
+	"bytes"
+	"fmt"
 	"log"
 	"os"
+	"strings"
 
 	"github.com/spf13/cobra"
+	"github.com/stormforger/cli/api/organisation"
+	"github.com/stormforger/cli/api/testcase"
 )
 
-// CompletionCmd is the command to generate bash/zsh completion
-var CompletionCmd = &cobra.Command{
-	Use:   "completion",
-	Short: "Create completion files for bash and zsh.",
-	Long: `Create shell completion code to tab-complete forge's commands.
-
-The generated completion files will be output in the current folder:
-- forge-completion-bash.sh
-- forge-completion-zsh.sh
-
-Zsh (experimental)
-------------------
-1. Rename forge-completion-zsh.sh to _forge and move it into a directory
-   that is part of your $fpath
-2. Re-initialize your completion files:
-   rm -f ~/.zcompdump; compinit
-
-Bash
-----
-To enable bash completion for forge:
-1. Edit your ~/.bash_profile and add a line like this:
-   source /path/to/forge-completion-bash.sh
-2. Start a new terminal session
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+$ source <(yourprogram completion bash)
+
+# To load completions for each session, execute once:
+Linux:
+  $ yourprogram completion bash > /etc/bash_completion.d/yourprogram
+MacOS:
+  $ yourprogram completion bash > /usr/local/etc/bash_completion.d/yourprogram
+
+Zsh:
+
+# If shell completion is not already enabled in your environment you will need
+# to enable it.  You can execute the following once:
+
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+# To load completions for each session, execute once:
+$ yourprogram completion zsh > "${fpath[1]}/_yourprogram"
+
+# You will need to start a new shell for this setup to take effect.
+
+Fish:
```

Comment:
the examples are missing powershell :)

Reply:
That's copy & paste from the generated command by cobra 🤷‍♂️

Reply:
ah ok, then good enough :)